en/app.ftl

```diff
@@ -397,6 +397,8 @@ profile-filter-category-option-relay-aliases-v2 = Random aliases
 profile-filter-category-option-domain-based-aliases-v2 = Custom aliases
 # Filter on Relay aliases that only forward critical emails.
 profile-filter-category-option-critical-only-aliases = Critical-only aliases
+# Filter on Relay aliases that block promotional emails. "Promo" is an English slang/shortened version of "Promotion".
+profile-filter-category-option-promo-blocking-aliases = Promo-blocking aliases
```

Comment:
I added a comment to address the use of the abbreviation in this label. Please let me know if there's additional context I can provide. 

cc/ @peiying2